content/resources/known-amendments.md

```diff
@@ -242,15 +242,15 @@ Introduces three new transaction types: CheckCreate, CheckCancel, and CheckCash,
 | Amendment    | Clawback |
 |:-------------|:---------|
 | Amendment ID | 56B241D7A43D40354D02A9DC4C8DF5C7A1F930D92A9035C4E12291B3CA3E1C2B |
-| Status       | Open for Voting |
+| Status       | Enabled |
```

Comment:
Could you fix the Japanese page as well?

Reply:
Thanks for catching that! Updated the Japanese page too.